src/components/table/table.tsx

```diff
@@ -8,7 +8,20 @@ import {
     Event,
     Host,
 } from '@stencil/core';
-import TabulatorTable from 'tabulator-tables';
+import {
+    TabulatorFull,
+    Tabulator,
+    Options as TabulatorOptions,
+    Sorter as TabulatorSorter,
+    SorterFromTable as TabulatorSorterFromTable,
+    SortDirection as TabulatorSortDirection,
+    ColumnComponent as TabulatorColumnComponent,
+    ColumnDefinition as TabulatorColumnDefinition,
+    RowComponent as TabulatorRowComponent,
+    OptionsData as TabulatorOptionsData,
+    OptionsPagination as TabulatorOptionsPagination,
```

Comment:
One of the styling issues that we face in the Webclient is because the pagination bar is not rendered anymore (in this updated version). 
<img width="517" alt="image" src="https://github.com/user-attachments/assets/495053b6-8f4b-415b-a1fd-7e36a2d25ec3" />